go/host/host.go

```diff
@@ -191,6 +212,19 @@ func (h *host) Start() error {
 	return nil
 }
 
+func (h *host) HandleBlock(block *types.Block) {
```

Comment:
would be good to add a comment here that the "host" object itself is a subscriber for L1 blocks